Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java

```diff
@@ -17,7 +17,9 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
+import javax.el.PropertyNotWritableException;
```

Comment:
Exception class is only used for / in documentation. Should we introduce really this dependency? Maybe we should use our own exception class.

This is used in many places but I will comment it only here. The change should be made on all places.